### PR TITLE
Quality and security improvements

### PR DESCRIPTION
**Pull request by jackpordi**
- Ran ESLint with the .eslintrc file I added
- Added [helmet](https://github.com/helmetjs/helmet) to express, which should improve security
- Did some manual fixes for uncaught errors
- Added 'use strict'

### CODE REVIEW

File: src/admin.js [hunk 1]
```diff
@@ -1,128 +1,119 @@
...
+      next()
+    } else {
+      res.redirect('/admin-login')
     }
-  }, (req,res) => {
+  }, (req, res) => {
     db.Sponsor.find((err, s) => {
-      res.render('admin', {sponsors: s}) 
+      if (err) {
+        logger.error('Error with admin login: ' + err)
```
Comment (jackpordi) [inline src/admin.js:39]: Check if this message is accurate please

File: auth/auth.js [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
-      session.login = true 
-      session.type = 'sponsor' 
-      session.user = user 
-      callback(true)
-    }else{
-      logger.info('sponsor ' + user +' has failed logging in') 
-      callback({member: false, err: 'Wrong username or password'}) 
+  db.Sponsor.find({ username: user }, (err, result) => {
+    if (err) {
+      return logger.error(err)
```
Comment (jackpordi) [inline auth/auth.js]: whoops, missed this one. What should the error message here be?

File: src/admin.js (hunk 1 above)
Comment (jackpordi) [reply src/admin.js:39]: @georgesoteriou

File: auth/auth.js (hunk 2 above)
Comment (jackpordi) [reply auth/auth.js]: @georgesoteriou

File: src/admin.js (hunk 1 above)
Comment (georgesoteriou) [reply src/admin.js:39]: 'Error with getting sponsors for admin portal: ' + error

File: auth/auth.js (hunk 2 above)
Comment (georgesoteriou) [reply auth/auth.js]: 'Unable to find sponsor: ' + err

File: src/admin.js [hunk 3]
```diff
@@ -1,128 +1,119 @@
...
-  }, (req,res) => {
-    var user = req.body.user 
+  }, (req, res) => {
+    var user = req.body.user
     var pass = req.body.pass
-    if (user === "docsoc" && pass === 'docsoc') {
-      req.session.docsoc = true 
-      res.redirect('/admin') 
-    }else{
+    if (user === 'docsoc' && pass === 'docsoc') {
```
Comment (georgesoteriou) [inline src/admin.js:21]: how do we secure this?

**Review by georgesoteriou — CHANGES_REQUESTED** [review]

**Pull request merged**